packages/@headlessui-react/src/utils/render.ts

```diff
@@ -57,7 +57,16 @@ export type PropsForFeatures<T extends RenderFeatures> = Expand<
   >
 >
 
-export function render<TFeature extends RenderFeatures, TTag extends ElementType, TSlot>({
+export function useRender() {
+  let mergeRefs = useMergeRefsFn()
+
+  return useCallback(
+    (args: Parameters<typeof render>[0]) => render({ mergeRefs, ...args }),
+    [render, mergeRefs]
```

Comment:
Since `render` isn't defined inside of a React hook it can be omitted from the dep array right? Or does ESLint complain? It obvs doesn't hurt anything being there just thinking out loud.

Reply:
Ah yep definitely can, but I include everything by default 🙈